frontend/src/app/pages/login-page/login-page.component.ts

```diff
@@ -78,7 +81,8 @@ export class LoginPageComponent implements OnInit {
                 console.log(userInfo);
                 if (userInfo) {
                     sessionStorage.setItem('userInfo', JSON.stringify(userInfo));
-                    this.routerService.navigate([appRoutes.HOME]);
+                    this.user = JSON.parse(sessionStorage.getItem('userInfo'));
```

Comment:
Can we do it once during Component initialization?

Reply:
No.